package solve

func SolveSecondTask() error {
	return nil
}
